src/items.c

```diff
@@ -0,0 +1,118 @@
+#include "items.h"
+#include <stddef.h>
+#include <math.h>
+#include <string.h>
+#include "test_items.h"
+
+
+int items_find_best_match(basket_item_s requested_item, store_s *store, found_item_s *found_destination, basket_item_s *missing_destination) {
+
+    double variance = 0.10; // TODO read from conf file
+
+
+    // Initialising the fields that will be compared against in the  search loop, in order to prevent garbage memory
+    // creating unexpected results.
+    found_item_s best_item = {
+        .count = 0,
+        .price_per_unit = -1, // Set to -1 to differentiate first run.
+    };
+
+    int item_found = 0;
+
+    for (int ite = 0; ite < store->items_count; ite++) {
```

Comment:
At a glance, naming this something other than simply 'i' suggests a special meaning, or possibly some variable set in another way, that doesn't just loop. We should probably stick with the normal conventions.